app/assets/stylesheets/issues.scss

```diff
@@ -0,0 +1,8 @@
+// Place all the styles related to the Issues controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.simple_form.Issue {
+  #Issue_description {
```

Comment:
Avoid using id selectors<br>Selector `Issue_description` should be written in lowercase with hyphens